src/drivers/intel/cavs/mn.c

```diff
@@ -19,10 +19,31 @@
 #define tracev_mn(__e, ...) \
 	tracev_event(TRACE_CLASS_MN, __e, ##__VA_ARGS__)
 
+/** \brief BCLKs can be driven by multiple sources - M/N or XTAL directly,
```

Comment:
nit-pick: 
the description can be improved a bit. Even in the case of M/N, the actual clock source can be XTAL, Audio cardinal clock (24.576) or 96 MHz PLL. The MN block is not really the source of clocks, but rather an intermediate component.

Reply:
Thanks, added this explanation to description.